sphinx/texinputs/Makefile_t

```diff
@@ -57,30 +73,28 @@ bz2: tar
 xz: tar
 	xz -9 -k $(ARCHIVEPREFIX)docs-$(FMT).tar
 
-# The number of LaTeX runs is quite conservative, but I don't expect it
-# to get run often, so the little extra time won't hurt.
-%.dvi: %.tex
-	$(LATEX) $(LATEXOPTS) '$<'
-	$(LATEX) $(LATEXOPTS) '$<'
-	$(LATEX) $(LATEXOPTS) '$<'
-	-$(MAKEINDEX) -s python.ist '$(basename $<).idx'
-	$(LATEX) $(LATEXOPTS) '$<'
-	$(LATEX) $(LATEXOPTS) '$<'
-
-%.pdf: %.tex
-	$(PDFLATEX) $(LATEXOPTS) '$<'
-	$(PDFLATEX) $(LATEXOPTS) '$<'
-	$(PDFLATEX) $(LATEXOPTS) '$<'
-	-$(MAKEINDEX) -s python.ist '$(basename $<).idx'
-	$(PDFLATEX) $(LATEXOPTS) '$<'
-	$(PDFLATEX) $(LATEXOPTS) '$<'
+{% if latex_engine == 'xelatex' -%}
+%.xdv: %.tex FORCE_MAKE
+{%- else -%}
+%.dvi: %.tex FORCE_MAKE
+{%- endif %}
+	$(LATEX) $(LATEXOPTS) "$<"
+
+{% if latex_engine == 'platex' -%}
+%.pdf: %.dvi FORCE_MAKE
+{%- else -%}
+%.pdf: %.tex FORCE_MAKE
+{%- endif %}
+	$(PDFLATEX) $(LATEXOPTS) "$<"
+
+{%- if latex_engine != 'xelatex' %}
 
 %.ps: %.dvi
-	dvips '$<'
+	dvips "$<"
+{%- endif %}
```

Comment:
- the quotes here have been changed due to Windows.
- it would be possible to provided ps target via ps2pdf from pdf